src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal sealed class AssemblyModel
```

Comment:
I will look into this in future PR.

---
In reply to: [101324384](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r101324384) [](ancestors = 101324384)